war/src/main/js/config-scrollspy.less

```diff
@@ -11,6 +11,7 @@
  * Widget styles
  */
 @import "widgets/config/tabbar";
+@import "widgets/add/addform";
```

Comment:
@tfennelly @recena So the config page and the add item page share quite a few, but not all style definitions. As an example, both pages have tabs. It sucks to have tab-styling repeated. Maybe shared mixins is the answer? My experience with Less is too small to have good instincts here, yet I have a lot of style changes I am trying to push through in a small window of time. I think it will make sense to circle back but not hang this merge.